config/postgres_password

```diff
@@ -1 +1 @@
-postgres
```

Comment:
Hopefully you got it figured out, but this actually isn't where you're supposed to change the postgres information - it should be in `autoscheduler/autoscheduler/config/postgres-info.json`. As such, could you remove these changes? This is also the reason that the backend tests are failing - these files are used to initialize the postgres DB in our docker container that runs the backend tests.

Reply:
Yeah I made the change sorry about that